src/arrays.js

```diff
@@ -1,61 +1,133 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length){
```

Comment:
This one needs a little more work. Our test cases are not large enough to actually test the full functionality of this function. For example, if the index was 45 and the array was only length of 5, we would be returning the 40th index of the array, which does not exist. See if you can refactor this using the `%` operator.

Reply:
I will add to our test cases for next time, but well done on passing the tests that we had set up. :)